src/libraries/System.Text.RegularExpressions/tests/UnitTests/System.Text.RegularExpressions.UnitTests.csproj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- xUnit2008 is about regexes and isn't appropriate in the test project for regexes -->
+    <!-- SYSLIB0036 is about obsoletion of regex members -->
```

Comment:
NIT: are we also missing a comment for CS0436?

Reply:
Actually, is that warning because we are referencing both the source code directly and the tests automatically including a reference to all of the framework so you get duplicate types? If so, I believe we have an easy way to exclude a reference there, let me find it real quick.

Reply:
Yeah you can remove this No Warn and instead add the following item into the existing ItemGroup:

```xml
<DefaultReferenceExclusion Include="System.Text.RegularExpressions" />
```

Reply:
Cool, thanks.